app/Http/Controllers/ProductController.php

```diff
@@ -17,12 +17,14 @@ public function index()
     {
         $products = Product::join('product_in_stocks', 'products.id', '=', 'product_in_stocks.product_id')
             ->join('product_media', 'products.id', '=', 'product_media.product_id')
-            ->select(['products.name', 'price', 'products.id', 'brand','media_link'])
+            ->select(['products.name', 'price', 'products.id', 'brand', 'media_link',])
+            ->where('product_media.type', '=', 'big image')
             ->get();
         $commeShoes = Product::join('product_in_stocks', 'products.id', '=', 'product_in_stocks.product_id')
             ->join('product_media', 'products.id', '=', 'product_media.product_id')
-            ->select(['products.name', 'price', 'products.id', 'brand','media_link'])
+            ->select(['products.name', 'price', 'products.id', 'brand', 'media_link',])
```

Comment:
```suggestion
            ->select(['products.name', 'price', 'products.id', 'brand', 'media_link'])
```
array trên 1 hàng ko cần , cuối cùng.